src/main/java/com/programmers/vouchermanagement/customer/repository/JdbcCustomerRepository.java

```diff
@@ -0,0 +1,119 @@
+package com.programmers.vouchermanagement.customer.repository;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import com.programmers.vouchermanagement.configuration.properties.file.FileProperties;
+import com.programmers.vouchermanagement.customer.domain.Customer;
+import com.programmers.vouchermanagement.customer.domain.CustomerType;
+import com.programmers.vouchermanagement.util.UUIDConverter;
+
+@Repository
+@Profile("jdbc")
+public class JdbcCustomerRepository implements CustomerRepository {
+    private static final RowMapper<Customer> customerRowMapper = (resultSet, i) -> mapToCustomer(resultSet);
+    public static final int SINGLE_DATA_FLAG = 1;
+
+    private final NamedParameterJdbcTemplate namedParameterJdbcTemplate;
+    private final String blacklistFilePath;
+
+    public JdbcCustomerRepository(NamedParameterJdbcTemplate namedParameterJdbcTemplate, FileProperties fileProperties) {
+        this.namedParameterJdbcTemplate = namedParameterJdbcTemplate;
+        this.blacklistFilePath = fileProperties.getCSVCustomerFilePath();
+        loadBlacklistToStorage();
+    }
+
+    private static Customer mapToCustomer(ResultSet resultSet) throws SQLException {
+        final UUID customer_id = UUIDConverter.from(resultSet.getBytes("customer_id"));
```

Comment:
자마 컨벤션에 맞게 적으면 어떨까요?

```suggestion
        final UUID customerId = UUIDConverter.from(resultSet.getBytes("customer_id"));
```

Reply:
꼼꼼했어야 했는데 놓쳤던 부분인 것 같습니다. 반영하겠습니다!